content/en/case-studies/gw-discov.md

```diff
@@ -55,81 +76,93 @@ made from warped spacetime.
-    [O(10^7) CPU hrs needed for binary merger analyses](https://youtu.be/7mcHknWWzNI)
-    spread on 6 dedicated LIGO clusters 
+and have tiny interaction with matter.  Processing and analyzing all of LIGO's
+data requires a vast computing infrastructure.After taking care of noise, which
+is billions of times of the signal, there is still very complex relativity
+equations and huge amounts of data which present a computational challenge:
+[O(10^7) CPU hrs needed for binary merger
+analyses](https://youtu.be/7mcHknWWzNI) spread on 6 dedicated LIGO clusters 
 
 * **Data Deluge**
 
     As observational devices become more sensitive and reliable, the challenges
-    posed by data deluge and finding a needle in a haystack rise multi-fold.
-    LIGO generates terabytes of data every day! Making sense of this data
-    requires an enormous effort for each and every detection. For example, the
-    signals being collected by LIGO must be matched by supercomputers against
-    hundreds of thousands of templates of possible gravitational-wave signatures.
+posed by data deluge and finding a needle in a haystack rise multi-fold.  LIGO
+generates terabytes of data every day! Making sense of this data requires an
+enormous effort for each and every detection. For example, the signals being
+collected by LIGO must be matched by supercomputers against hundreds of
+thousands of templates of possible gravitational-wave signatures. 
 
 * **Visualization**
 
-    Once the obstacles related to understanding Einstein’s equations well
-    enough to solve them using supercomputers are taken care of, the next big
-    challenge was making data comprehensible to the human brain. Simulation
-    modeling as well as  signal detection requires effective visualization
-    techniques.  Visualization also plays a role in lending more credibility
-    to numerical relativity in the eyes of pure science aficionados, who did
-    not give enough importance to numerical relativity until imaging and
-    simulations made it easier to comprehend results for a larger audience.
-    Speed of complex computations and rendering, re-rendering images and
-    simulations using latest experimental inputs and insights can be a time
-    consuming activity that challenges researchers in this domain.
-
-{{< figure src="/images/content_images/cs/gw_strain_amplitude.png" class="fig-center" alt="gravitational waves strain amplitude" caption="**Estimated gravitational-wave strain amplitude from GW150914**" attr="(**Graph Credits:** Observation of Gravitational Waves from a Binary Black Hole Merger, ResearchGate Publication)" attrlink="https://www.researchgate.net/publication/293886905_Observation_of_Gravitational_Waves_from_a_Binary_Black_Hole_Merger" >}}
+    Once the obstacles related to understanding Einstein’s equations well enough
+to solve them using supercomputers are taken care of, the next big challenge was
+making data comprehensible to the human brain. Simulation modeling as well as
```

Comment:
```suggestion
Simulation modeling as well as
```
This statement misses the mark a bit and is not really necessary. I agree that highlighting the role of visualization is nice though.